python/tests/test_tree_stats.py

```diff
@@ -254,9 +254,13 @@ class PythonNodeStatCalculator(object):
     def __init__(self, tree_sequence):
         self.tree_sequence = tree_sequence
 
-    def divergence(self, X, Y, windows=None):
+    def divergence(self, sample_sets, indices=None, windows=None):
```

Comment:
I switched the interface on all these testing methods to mirror the real one (easier testing).